src/site/content/en/blog/godaddy/index.md

```diff
@@ -0,0 +1,97 @@
+
+This generic image behavior allowed us to pre-calculate the final image height during server-side rendering, based on viewport width and image aspect ratio. This resulted in HTML markup with vertical space appropriately reserved for the final image. The improvement was particularly observable on mobile devices, since images are rendered to the full span of mobile viewports.
+
+Certain components on our customers sites have dynamic content (eg. list of external customer reviews) and could not be converted to pure CSS to leverage performance benefits of server side rendering. These are difficult areas to improve layout shifts because content (thus height) will vary. In those cases, we **wrapped the component in a container with a `min-height` applied**, predetermined based on observation of the average height for each of the specific components. The min-height is removed once the inner dynamic component is done rendering. While not perfect, this solution allowed us to reduce layout shift by a lot.
+
+While focusing on CLS improvements, we continued to work on LCP. On many websites, images are the biggest culprit contributing to LCP and for us it was an obvious area of improvement. We had made improvements to **lazy load images** using [`IntersectionObserver`](https://developer.mozilla.org/docs/Web/API/IntersectionObserver) but realized that image sizes were not set in the most optimal way for each breakpoint (mobile, tablet, desktop, large desktop), so we updated our image generation code to clamp and scale images per breakpoint and then again scale resolution based on pixel density. As an example, this reduced the size of a specific large image from 192 kb to 102 kb.
+
+In order to quickly render websites on devices with poor network connections, **we added code to dynamically scale down image quality based on connection speed**. This can easily be done using the `downlink` property returned by [`navigator.connection`](https://developer.mozilla.org/docs/Web/API/Navigator/connection). We apply URL-based query parameters to reduce image quality through our asset API based on those network conditions.
+
+A number of sections of our customer sites are dynamically loaded, so in order to get a head start on these requests, and since we know what sections will be rendered on a given site when it's published, **we used the [`rel=preconnect` resource hint](https://www.w3.org/TR/resource-hints/#preconnect)** to initialize the connection and necessary handshakes. We also use resource hints to load fonts and other important resources quickly.
```

Comment:
```suggestion
A number of sections of our customer sites are dynamically loaded. Since we know what sections will be rendered on a given site when it's published, we used the [`rel=preconnect` resource hint](https://www.w3.org/TR/resource-hints/#preconnect) to initialize the connection and necessary handshakes ahead of time. We also use resource hints to load fonts and other important resources quickly.
```

Reply:
Could link to web.dev docs on resource hints here